cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/query/QueryStates.kt

```diff
@@ -64,29 +64,43 @@ data object NotYetEvaluated : DecisionState()
 /**
  * Wraps the given [QueryTree] in a new [Decision] object by checking if the value is `true` or
  * `false` and based on the [de.fraunhofer.aisec.cpg.assumptions.Assumption.status] of all
- * [QueryTree.assumptions] (i.e., it checks if all are [AssumptionStatus.Accepted] or if some are
- * [AssumptionStatus.Rejected] or [AssumptionStatus.Undecided]).
+ * [QueryTree.collectAssumptions] (i.e., it checks if all are [AssumptionStatus.Accepted] or if some
+ * are [AssumptionStatus.Rejected] or [AssumptionStatus.Undecided]).
  */
 context(TranslationResult)
 fun QueryTree<Boolean>.decide(): Decision {
     val statues = this@TranslationResult.assumptionStatuses
-    // TODO(kweiss): Use status to set status
+    // The assumptions need to be collected, as they are located at the respective construct they
+    // are placed on and only
```

Comment:
odd formatting

Reply:
Spotless. Apparently somethings one has to do manually.